docs/Compile.rst

```diff
@@ -361,9 +361,10 @@ To install Chocolatey and the required dependencies:
 
 * Paste in the Chocolatey install command and hit enter
 * Close this ``cmd.exe`` window and open another Admin ``cmd.exe`` in the same way
-* Run the following command::
+* Run the following commands::
 
-    choco install git cmake strawberryperl -y
+    choco install git cmake.portable strawberryperl -y
+    cpanm XML::LibXML
```

Comment:
@moversti I have just done a compile on a fresh Windows 7 installation, just to be 100% certain of what I was saying on cpan.

I can confirm that for certain, cpanm step is not required.  StrawberryPerl provides XML::LibXML and XML::LibXSLT already, and the provided versions are fine to run the Perl needed for DFHack compilation.

I am wondering if maybe you have a second Perl install on your system, which is higher in PATH than StrawberryPerl?  So maybe you thought you had to run cpanm because you are in fact not using StrawberryPerl, and your other Perl install did not have XML::LibXML installed.

Could you let me know if you are going to re-commit to this PR to remove that.  If not, I will open a new PR to change cmake -> cmake.portable which definitely is required (thanks for raising that!)

Also I am going to update the Chocolatey instructions slightly, to mention that it may not be necessary to re-open a new cmd.exe window. I found in my Windows 7 testing just now that I did not have to, despite Chocolatey's warnings saying that I almost certainly would have to.  Maybe this warning applies only to Windows XP or other, older Windows.